internal/npm_install/generate_build_file.ts

```diff
@@ -255,8 +277,8 @@ function generatePackageBuildFiles(pkg: Dep) {
   const nodeModulesPkgDir = `node_modules/${pkg._dir}`;
   // Check if the current package dep dir is a symlink (which happens when we
   // install a node_module with link:)
-  const isPkgDirASymlink =
-      fs.existsSync(nodeModulesPkgDir) && fs.lstatSync(nodeModulesPkgDir).isSymbolicLink();
+  const isPkgDirASymlink = await fs.lstat(nodeModulesPkgDir)
+    .then(stat => stat.isSymbolicLink()).catch(() => false)
```

Comment:
Why is the `catch` necessary unlike previously?

Reply:
If nodeModulesPkgDir is not existed, `fs.lstat` throws an error, so I catch the error instead of call existsSync before, It may reduce syscall.